src/views/v2/tickets.php

```diff
@@ -72,6 +72,7 @@ class="tribe-tickets"
 
 	</form>
 
-	<?php $this->template( 'v2/modal/target' ); ?>
+	// @todo Convert this into an action.
```

Comment:
This comment is in plain HTML, it would be displayed in the page.

Reply:
Yeah that became painfully clear when I saw the comment in the diff :-D